internal/common/metrics/constants.go

```diff
@@ -52,7 +52,8 @@ const (
 
 	LocalActivityTotalCounter           = TemporalMetricsPrefix + "local_activity_total"
 	LocalActivityCanceledCounter        = TemporalMetricsPrefix + "local_activity_canceled"
```

Comment:
Should we also have `local_activity_execution_cancelled`?

Reply:
yeah that's missing to I'll add in this PR